### DISCUSSION TITLE
Insufficient disk space

### DISCUSSION THREAD

**Discussion by alexis-mosquerac** [datalust/seq-tickets#1169] [Help]
Hi team.
I'm receiving this message: "the disk holding seq event storage has insufficient space; arriving events will be dropped" but the disk has 141GB free and the "Minimum free storage space" setting is 128MB.  I'm using Seq 4.2.476.  Could you please help me to setup a propper disk limit?

**Comment by nblumhardt** [comment]
Hi Alexis, thanks for getting in touch. I've replied via the duplicate post on our old discussion forum.
